src/components/dashboard/index.js

```diff
@@ -172,7 +174,7 @@ export const DashboardItem = (props) => {
     return (
         <Card
             className={classes.dashboardCard}
-            id={buildID(ids.ITEM_BASE, cardID)}
+            id={makeID(`${ids.ITEM}.${cardID}`)}
```

Comment:
With the above refactor to `makeID`, instead of trying to construct the same `parent.child` pattern here, you could then have this be `makeID(ids.ITEM, cardID)`. That way we're relying on `build`, or `buildID` in this case, to maintain that pattern.

Reply:
Thanks for catching this.